.github/workflows/ci.yml

```diff
@@ -8,18 +8,21 @@ on:
       - "*"
 
 jobs:
-  build:
+  linux-build:
```

Comment:
Do we need to replicate the jobs? 
Can we use Github Actions matrix for platforms[1]

[1] https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/precommit.yml#L6

Reply:
That's a good idea, I'll go ahead and make this change